src/Mono.Android.Export/Mono.Android.Export.csproj

```diff
@@ -4,70 +4,33 @@
   <Import Project="..\..\Configuration.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>monoandroid10;$(DotNetTargetFramework)</TargetFrameworks>
+    <TargetFrameworks>$(DotNetTargetFramework)</TargetFrameworks>
```

Comment:
I think we may be able to skip some extra outer/inner build processing by changing this to `TargetFramework` now, though I'm not sure if it makes much of a difference.

Reply:
I like this suggestion, and it probably applies to a lot of projects.  I'll include this in some future cleanups.  👍